modules/hcp-scale-the-nodepool.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-scale-the-nodepool_{context}"]
+== Scaling the node pool
+
+After your agents are approved, you can scale the node pools. The `agentLabelSelector` value that is configured in the node pool ensures that only matching agents are added to the cluster. This also helps scale down the node pool. To remove specific architecture nodes from the cluster, scale down the corresponding node pool.
+
+.Procedure
+
+. Scale the node pool by running the following command:
++
+[source,terminal]
+----
+$ oc -n <clusters_namespace> scale nodepool <nodepool_name> --replicas 2
+----
++
+[NOTE]
+====
+The Cluster API agent provider picks two agents randomly to assign to the hosted cluster. These agents pass through different states and then join the hosted cluster as {product-title} nodes. The various agent states are `binding`, `discovering`, `insufficient`, `installing`, `installing-in-progress`, and `added-to-existing-cluster`.
+====
+
+[id="hcp-scale-nodepool-verifying_{context}"]
+=== Verifying the scaling of the node pools
```

Comment:
This is nested pretty deep in the content. Also, because the content above this counts as "a procedure," this content would technically need to be in a different module, as we only allow one procedure per module. This either needs split out, or it needs to be formatted as a more traditional '.Verification' section we do in procedures (in this scenario, you'd remove lines 25-26 and replace them with .Verification):

```suggestion
.Verification
```